src/adapters/lib/SnapshotSourceLib.sol

```diff
@@ -1,37 +1,36 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.17;
 
-import {DiamondRootOval} from "../../DiamondRootOval.sol";
-
 /**
- * @title SnapshotSource contract to be used in conjunction with a source adapter that needs to snapshot historic data.
+ * @title SnapshotSourceLib library to be used by a source adapter that needs to snapshot historic data.
  */
-abstract contract SnapshotSource is DiamondRootOval {
+library SnapshotSourceLib {
     // Snapshot records the historical answer at a specific timestamp.
     struct Snapshot {
```

Comment:
no reason really to snapshot round ID I guess? especially given any source that has the notion of a round ID also has history?